cmd/cli/namespace_test.go

```diff
@@ -114,6 +115,49 @@ var _ = Describe("Running the namespace add command", func() {
 			})
 		})
 
+		Context("Given one namespace as an arg with sidecar injection disable", func() {
```

Comment:
Change to:

"Given one namespace as an arg with sidecar injection explicitly set to false"